src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
```

Comment:
no needs to use template string
```suggestion
      My name is {person.name}
```